shorebird_code_push/example/lib/main.dart

```diff
@@ -31,6 +31,7 @@ class _MyHomePageState extends State<MyHomePage> {
   late final bool _isUpdaterAvailable;
   var _isCheckingForUpdates = false;
   Patch? _currentPatch;
+  UpdateTrack? _currentTrack;
```

Comment:
Need to also add a bool to track the initialization state of the track

Reply:
"track the initialization state" = `hasUpdateTrackBeenSet`?